modules/abstract-utxo/src/recovery/crossChainRecovery.ts

```diff
@@ -84,7 +84,10 @@ export async function getWallet(
  * @param wallet
  * @return wallet pubkeys
  */
-async function getWalletKeys(recoveryCoin: AbstractUtxoCoin, wallet: IWallet | WalletV1): Promise<RootWalletKeys> {
+export async function getWalletKeys(
+  recoveryCoin: AbstractUtxoCoin,
+  wallet: IWallet | WalletV1
+): Promise<RootWalletKeys> {
```

Comment:
+1

Reply:
I was thinking the same thing but IMO it can stay there. There isnt really a better place to put it in `AbtractUtxo` and it gets exported to the highest level anyway